clients/privacy-center/config/config.json

```diff
@@ -8,11 +8,11 @@
     {
       "policy_key": "default_access_policy",
       "icon_path": "/download.svg",
-      "title": "Access your data",
-      "description": "We will provide you a report of all your personal data.",
+      "title": "Access your data (email or phone number)",
+      "description": "Please provide either an email or a phone number. We will provide you a report of all your personal data.",
       "identity_inputs": {
         "name": "optional",
-        "email": "required",
```

Comment:
By making the default lack any `required` inputs, we create a bit of a problem where it's possible to just submit the form blank. That throws an error message at the user that isn't very helpful.